web-transport-quinn/web/client.js

```
@@ -2,10 +2,7 @@
 import fingerprintHex from 'bundle-text:../cert/localhost.hex';
 
 // Convert the hex to binary.
-let fingerprint = [];
-for (let c = 0; c < fingerprintHex.length - 1; c += 2) {
-    fingerprint.push(parseInt(fingerprintHex.substring(c, c + 2), 16));
-}
+const fingerprint = Uint8Arra.fromHex(fingerprint);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)